systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -45,7 +76,9 @@ TBA
 | ApiErrorReporter                            | API response        | A component for API modules that handles error reporting                                                                                            |
 | ResultBuilder                               | API response        | Builder of MediaWiki ApiResult objects with various convenience functions for adding Wikibase concepts and result parts to results in a uniform way |
 
-### Special Merge Items
+### Special Pages
```

Comment:
Per my comment above I also think this should be a level 2 heading.

Reply:
done